src/Lykke.Service.HFT/Modules/ClientsModule.cs

```diff
@@ -43,7 +41,7 @@ protected override void Load(ContainerBuilder builder)
         }
 
         // TODO Remove when all client libraries support new logging framework
-        private class LogWrapper : DynamicObject, ILog
+        private class LogWrapper : ILog
```

Comment:
Please, give me the list of client libraries, which not supports ILogFactory yet.

Reply:
Lykke.Service.FeeCalculator.Client
Lykke.Service.OperationsHistory.Client